saml2/saml2-service-provider/src/test/java/org/springframework/security/saml2/provider/service/registration/OpenSamlRelyingPartyRegistrationBuilderHttpMessageConverterTests.java

```diff
@@ -54,11 +54,11 @@
 	private static final String SINGLE_SIGN_ON_SERVICE_TEMPLATE = "<md:SingleSignOnService Binding=\"urn:oasis:names:tc:SAML:2.0:bindings:HTTP-Redirect\" "
 			+ "Location=\"sso-location\"/>";
 
-	private OpenSamlRelyingPartyRegistrationBuilderHttpMessageConverter converter;
+	private OpenSamlAssertingPartyMetadataConverter converter;
```

Comment:
This is just a reminder to add some file-based tests as I imagine you are just looking for initial feedback right now.

Reply:
Changed tests to handle an input stream instead of an http message